src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,64 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer zaz = manufacturerService.create(new Manufacturer("Zaz", "Ukraine"));
+        Manufacturer dacia = manufacturerService.create(new Manufacturer("Dacia", "Romania"));
+        Manufacturer seat = manufacturerService.create(new Manufacturer("Seat", "Spain"));
```

Comment:
`Manufacturer` variables also can be final because its never change in the code below.